docs/newbs.md

```diff
@@ -6,10 +6,11 @@ Not sure if your keyboard can run QMK? If it's a mechanical keyboard you built y
 
 ## Overview
 
-There are 6 main sections to this guide:
+There are 7 main sections to this guide:
 
 * [Getting Started](newbs_getting_started.md)
-* [Building Your First Firmware](newbs_building_firmware.md)
+* [Building Your First Firmware using the command line](newbs_building_firmware.md)
+* [Building Your First Firmware using the online GUI](newbs_building_firmware_configurator.md)
```

Comment:
I'd prefer using the actual name.

```suggestion
* [Building Your First Firmware using the QMK Configurator](newbs_building_firmware_configurator.md)
```

Reply:
I will disagree here because people who would be interested in an online graphical tool would not know that it's called QMK Configurator.